src/components/Toaster/component.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useCallback, useMemo } from 'react';
+import { Slide, ToastContainer } from 'react-toastify';
+
+import { Testable, useBuildTestId } from '../../modules/test-ids';
+import { Icon } from '../Icon';
+
+import { StyledButton, Styles } from './styled';
+
+export type Props = Omit<
+  React.ComponentPropsWithoutRef<typeof ToastContainer>,
+  'closeButton' | 'hideProgressBar' | 'transition'
+> &
+  Testable;
+
+export const Component = ({
+  position = 'bottom-center',
+  'data-testid': testId,
+  ...otherProps
+}: Props) => {
+  const { buildTestId } = useBuildTestId({ id: testId });
+
+  const close = useCallback(
+    ({ closeToast }: { closeToast: any }) => (
```

Comment:
Hmm can we have a type for `closeToast`?

Reply:
Done!